tests/auth/package.json

```diff
@@ -13,7 +13,10 @@
     "clean": "npx rimraf ./dist",
     "build": "npx tsc",
     "start": "node .",
-    "dev": "npx nodemon -w \"./src/**\" -e ts --exec \"node -r ts-node/register -r dotenv/config ./src/index.ts\""
+    "dev": "npx nodemon -w \"./src/**\" -e ts --exec \"node -r ts-node/register -r dotenv/config ./src/index.ts\"",
+    "dev:teamsfx": "NODE_OPTIONS='--inspect=9239' npx env-cmd -f .env npm run dev",
```

Comment:
not a huge deal since these are tests, but remember this method of setting env vars doesn't work in windows.

Reply:
You're right. I used the cli and I guess it wasn't updated. Will fix